### PR TITLE
Add name in relu and log API.

### PR DESCRIPTION
**Pull request by qingqing01**
Fix https://github.com/PaddlePaddle/Paddle/issues/12437
Fix https://github.com/PaddlePaddle/models/issues/1068

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -5108,7 +5108,7 @@ def log(x):
     return out
 
 
-def relu(x):
+def relu(x, name=None):
```
Comment (qingqing01) [inline python/paddle/fluid/layers/nn.py:5112]: 这里没有name的不兼容，是上次大量修改API doc时引入的。

**Review by luotao1 — APPROVED** [review]
LGTM

**Review by wanghaoshuang — APPROVED** [review]
LGTM.

**Review by panyx0718 — APPROVED** [review]

**Review by reyoung — APPROVED** [review]
Excellent. Sorry for reviewing it late.

**Pull request merged**